### ISSUE TITLE
 As a user I want a CTS Work URN (without a passage reference) to resolve to a list of URLs serving editions and translations of this work.

### ISSUE THREAD

**Issue by balmas** [rpidproject/cts-handles#2]
As a user I want a CTS Work URN (without a passage reference) to resolve to a list of URLs serving editions and translations of this work.

**Comment by amir-zeldes** [comment]
Shouldn't there be some way of encoding distinct editions in the URN itself? If so and this is just a substring, then probably yes.

**Comment by balmas** [comment]
yes, the URN has an edition component.  The requirement is is related to resolving requests for 'notional' or 'abstract' work references that exclude the edition component.
